test/Microsoft.ML.Tests/Transformers/CategoricalTests.cs

```diff
@@ -101,6 +108,30 @@ public void CategoricalOneHotEncoding()
             Done();
         }
 
+        [Fact]
+        public void CategoricalOneHotEncodingVector()
+        {
+            var data = new[] {
+                new TestClassWithLabel() { A = 301, B = 2000, Label = true },
+                new TestClassWithLabel() { A = 450, B = 3000, Label = true },
+                new TestClassWithLabel() { A = -300, B = 4000, Label = true },
+                new TestClassWithLabel() { A = 300, B = 2000, Label = false },
+                new TestClassWithLabel() { A = 115, B = 2000, Label = false },
+                new TestClassWithLabel() { A = 115, B = 2000, Label = false }};
+
+            var mlContext = new MLContext();
+            var dataView = mlContext.Data.LoadFromEnumerable(data);
+            var pipe = mlContext.Transforms.Conversion.ConvertType("A", outputKind: DataKind.Single)
+        .Append(mlContext.Transforms.Conversion.ConvertType("B", outputKind: DataKind.Single))
```

Comment:
(nit) the formatting here is off.  The `.Append` should be to the right of the line above it, for example:

```C#
            var pipe = mlContext.Transforms.Conversion.ConvertType("A", outputKind: DataKind.Single)
                .Append(mlContext.Transforms.Conversion.ConvertType("B", outputKind: DataKind.Single))
                .Append(mlContext.Transforms.Concatenate("Features", new string[] { "A", "B" }))
```

Reply:
DOne.